MCP HTTPS, release notes and policy merging (#422)
kgateway-dev/kgateway.dev: content/docs/main/reference/release-notes.md

* [Consistent hashing]({{< link-hextra path="/traffic-management/session-affinity/consistent-hashing/" >}}): Set up soft session affinity between a client and a backend service by using consistent hashing algorithms. 
* [Session persistence]({{< link-hextra path="/traffic-management/session-affinity/session-persistence/" >}}): Set up “strong” session affinity or sticky sessions to ensure that traffic from a client is always routed to the same backend instance for the duration of a session.

#### Enhanced retries and timeout capabilities {#retries-timeouts}

You can now set the following retries and timeouts for your Envoy-based gateway proxies:
* [Request retries]({{< link-hextra path="/resiliency/retry/retry/" >}})
* [Request timeouts]({{< link-hextra path="/resiliency/timeouts/request/" >}})
* [Per-try timeouts]({{< link-hextra path="/resiliency/retry/per-try-timeout/" >}})
* [Idle timeouts]({{< link-hextra path="/resiliency/timeouts/idle/" >}})
* [Idle stream timeouts]({{< link-hextra path="/resiliency/timeouts/idle-stream/" >}})

#### Passive health checks with outlier detection {#outlier-detection}

> Configure passive health checks and remove unhealthy hosts from the load balancing pool with an outlier detection policy. An outlier detection policy sets up several conditions, such as retries and ejection percentages, that kgateway uses to determine if a service is unhealthy. In case an unhealthy service is detected, the outlier detection policy defines how the service is removed from the pool of healthy destinations to send traffic to. For more information, see [Outlier detection]({{< link-hextra path="/resiliency/outlier-detection/" >}}).

Reviewer: ```suggestion
Configure passive health checks and remove unhealthy hosts from the load balancing pool with an outlier detection policy. An outlier detection policy sets up several conditions, such as retries and ejection percentages, that kgateway uses to determine if a service is unhealthy. When an unhealthy service is detected, the outlier detection policy defines how the service is removed from the pool of healthy destinations to send traffic to. For more information, see [Outlier detection]({{< link-hextra path="/resiliency/outlier-detection/" >}}).
```
